modules/rosa-using-spot-instance.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-using-spot-instance_{context}"]
+= Adding a machine pool with spot instances to a cluster
+
+After a machine pool is created, you can configure additional machine pools with spot instances.
+
+.Procedure
+
+ifdef::openshift-rosa[]
+. To configure a machine pool with a spot instance, enter the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --help | grep spot
+  # Add a machine pool with spot instances to a cluster
+  rosa create machinepool -c mycluster --name=mp-1 --replicas=2 --instance-type=r5.2xlarge --use-spot-instances \
+    --spot-max-price=0.5
+      :
+      --spot-max-price string   Max price for spot instance. If empty use the on-demand price. (default "on-demand")
+      --use-spot-instances      Use spot instances for the machine pool.
+----
+endif::[]
+
+////
+ifdef::openshift-dedicated[]
+To configure a machine pool with a spot instance:
+
+. Select the *Machine pools* tab.
+. Click the *Add machine pool* button on your OSD cluster page in OCM.
+. Add the machine pool with the spot instance by checking the *Use Amazon EC2 Spot Instances* check box.
```

Comment:
Lack of the steps for price settings and the last step click `Add machine pool` button.